mkdocs.yml

```diff
@@ -45,7 +45,10 @@ nav:
         - Usage: tm/usage.md
         - Registry: tm/registry.md
         - Serverless Manifest: tm/serverless.md
-    - Installation: guides/installation.md
+    - Installation:
+      - guides/installation.md
```

Comment:
Sorry for my ignorance, but that format is valid for Mkdocs, right?
The ones below are different.

Reply:
yes, it depends if there is a page for the default section or not.